ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplDumpTask.java

```diff
@@ -751,18 +767,19 @@ private void fetchFailoverMetadata(Hive hiveDb) throws HiveException, IOExceptio
       work.setFailoverMetadata(fmd);
       return;
     }
-    List<Long> txnsForDb = getOpenTxns(getTxnMgr().getValidTxns(excludedTxns), work.dbNameOrPattern);
+    HiveTxnManager hiveTxnManager = getTxnMgr();
```

Comment:
This call is not that costly, and if we use a static import for `ReplUtils.getOpenTxns` we would have a smaller change which is easier to review